test/expected/drop_rename_hypertable.out

```diff
@@ -317,32 +317,178 @@ Check constraints:
+ 26 | _timescaledb_internal | _hyper_1_4_chunk | 26-two_Partitions_timeCustom_series_bool_idx | newschema        | two_Partitions_timeCustom_series_bool_idx | CREATE INDEX "26-two_Partitions_timeCustom_series_bool_idx" ON _timescaledb_internal._hyper_1_4_chunk USING btree ("timeCustom" DESC NULLS LAST, series_bool) WHERE (series_bool IS NOT NULL)
+  6 | _timescaledb_internal | _hyper_1_1_chunk | 6-two_Partitions_timeCustom_device_id_idx    | newschema        | two_Partitions_timeCustom_device_id_idx   | CREATE INDEX "6-two_Partitions_timeCustom_device_id_idx" ON _timescaledb_internal._hyper_1_1_chunk USING btree ("timeCustom" DESC NULLS LAST, device_id)
+ 13 | _timescaledb_internal | _hyper_1_2_chunk | 13-two_Partitions_timeCustom_device_id_idx   | newschema        | two_Partitions_timeCustom_device_id_idx   | CREATE INDEX "13-two_Partitions_timeCustom_device_id_idx" ON _timescaledb_internal._hyper_1_2_chunk USING btree ("timeCustom" DESC NULLS LAST, device_id)
+ 20 | _timescaledb_internal | _hyper_1_3_chunk | 20-two_Partitions_timeCustom_device_id_idx   | newschema        | two_Partitions_timeCustom_device_id_idx   | CREATE INDEX "20-two_Partitions_timeCustom_device_id_idx" ON _timescaledb_internal._hyper_1_3_chunk USING btree ("timeCustom" DESC NULLS LAST, device_id)
+ 27 | _timescaledb_internal | _hyper_1_4_chunk | 27-two_Partitions_timeCustom_device_id_idx   | newschema        | two_Partitions_timeCustom_device_id_idx   | CREATE INDEX "27-two_Partitions_timeCustom_device_id_idx" ON _timescaledb_internal._hyper_1_4_chunk USING btree ("timeCustom" DESC NULLS LAST, device_id)
+  7 | _timescaledb_internal | _hyper_1_1_chunk | 7-two_Partitions_timeCustom_idx              | newschema        | two_Partitions_timeCustom_idx             | CREATE INDEX "7-two_Partitions_timeCustom_idx" ON _timescaledb_internal._hyper_1_1_chunk USING btree ("timeCustom" DESC)
+ 14 | _timescaledb_internal | _hyper_1_2_chunk | 14-two_Partitions_timeCustom_idx             | newschema        | two_Partitions_timeCustom_idx             | CREATE INDEX "14-two_Partitions_timeCustom_idx" ON _timescaledb_internal._hyper_1_2_chunk USING btree ("timeCustom" DESC)
+ 21 | _timescaledb_internal | _hyper_1_3_chunk | 21-two_Partitions_timeCustom_idx             | newschema        | two_Partitions_timeCustom_idx             | CREATE INDEX "21-two_Partitions_timeCustom_idx" ON _timescaledb_internal._hyper_1_3_chunk USING btree ("timeCustom" DESC)
+ 28 | _timescaledb_internal | _hyper_1_4_chunk | 28-two_Partitions_timeCustom_idx             | newschema        | two_Partitions_timeCustom_idx             | CREATE INDEX "28-two_Partitions_timeCustom_idx" ON _timescaledb_internal._hyper_1_4_chunk USING btree ("timeCustom" DESC)
+(28 rows)
+
+\d _timescaledb_internal._hyper_1_1_chunk
```

Comment:
I am not sure what these are testing? That the inherits is right? Can you put a comment about what I should verify?